Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/common/use-route-url-params.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { createContext, useCallback, useContext, useMemo, useState } from 'react';
+
+import { Anything } from '../../common/common-types';
+import { IHaveChildrenProps } from '../../components/common/Props';
+
+interface IRouteUrlParamsContext {
+    state: {
+        params: any;
```

Comment:
can this be type of Anything instead of any

Reply:
This breaks the Heading of ContestResultPage